src/Registry.ts

```diff
@@ -29,31 +30,55 @@ export interface RegistryEvents extends BaseEventedEvents {
 /**
  * Widget Registry Interface
  */
-export interface Registry {
+export interface RegistryInterface {
```

Comment:
We have traditionally not mentioned `Interface` in our interfaces.  Why the change?

Reply:
Did it have something to do with name merging?  Could this be a use case for an abstract class and having an `AbstractRegistry` as an export?

Reply:
The reason was, that without a different name for the interface TS didn't actually error if the class didn't implement it! We've used a suffix of `Interface` in some of the interfaces exported from `interfaces.d.ts` which was the motivation.

I don't think `Abstract` works because the registry isn't actually abstract?

Reply:
Makes sense.  Thanks.